content/reference/admin/private_locations/configuration/azure/index.md

```diff
@@ -24,6 +24,10 @@ Check Azure documentation pages to find these values:
 * [Client id and secret](https://learn.microsoft.com/en-us/answers/questions/834401/hi-i-want-my-client-id-and-client-secret-key)
 * [Subscription id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id)
 
+{{< alert warning >}}
+Azure private locations rely on [cloud-init](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init) integration.
+{{< /alert >}}
```

Comment:
Shouldn't this be worded in a way which explains _why_ it matters to the users? (i.e., "if you configure a custom image, make sure it supports cloud-init" - I assume that's what we're trying to tell the users here?)

Reply:
Nice! 👍 I removed the warning from azure because we do not propose custom images at the moment (so the warning is just confusing). But added the why to aws cloud init warn.